action.yml

```diff
@@ -26,9 +26,15 @@ inputs:
     required: false
     default: false
   target:
-    description: 'Auto-merge on major, minor, patch updates based on Semantic Versioning'
+    description: 'Auto-merge on major, minor, patch updates based on Semantic Versioning for both dev and production dependencies'
```

Comment:
are we by any chance leaving out other types of dependencies? there is more than dev and prod. it's fine if we don't have inputs for each of them, but we should probably not specify the list here because this input covers everything, not just dev and prod, but also, e.g. peer

Reply:
also, this descripion here is a bit misleading, because it only specifies major, minor and patch, but we support more. let's change the description of this input and the new inputs to be more generic, the full details are in the docs